stdlib/nntplib.pyi

```diff
@@ -34,11 +35,13 @@ def decode_header(header_str: str) -> str: ...
 
 _list = list  # conflicts with a method named "list"
 
-class _NNTPBase:
-    encoding: str
-    errors: str
+class NNTP:
+    encoding: ClassVar[str]
+    errors: ClassVar[str]
```

Comment:
These might make more sense if typed as instance attributes, so that you don't have to subclass `NNTP` or `NNTP_SSL` just to change them.